torchao/testing/model_architectures.py

```diff
@@ -11,14 +11,53 @@
 import torch.nn.functional as F
 
 
-# TODO: Refactor torchao and tests to use these models
-class ToyLinearModel(torch.nn.Module):
-    def __init__(self, k=64, n=32, dtype=torch.bfloat16):
+class ToySingleLinearModel(torch.nn.Module):
+    """Single linear for m * k * n problem size"""
+
+    def __init__(
+        self, m=64, n=32, k=64, has_bias=False, dtype=torch.float, device="cuda"
+    ):
+        super().__init__()
+        self.m = m
+        self.dtype = dtype
+        self.device = device
+        self.linear = torch.nn.Linear(k, n, bias=has_bias).to(
+            dtype=self.dtype, device=self.device
+        )
+
+    def example_inputs(self):
+        return (
+            torch.randn(
+                self.m, self.linear.in_features, dtype=self.dtype, device=self.device
+            ),
+        )
+
+    def forward(self, x):
+        x = self.linear(x)
+        return x
+
+
+class ToyMultiLinearModel(torch.nn.Module):
+    def __init__(self, m=512, n=256, k=128, has_bias=True):
```

Comment:
I feel `m`, `n`, `k` should be required actually

also do we need 3 linears? can this be 2 linears and rename to `TwoLinearModel` to make it clearer?